portality/view/doaj.py

```diff
@@ -58,13 +58,21 @@ def news():
     news = blog.News.latest(app.config.get("NEWS_PAGE_NEWS_ITEMS", 20))
     return render_template('doaj/news.html', news=news, blog_url=app.config.get("BLOG_URL"))
 
+@blueprint.route("/widgets_demo")
```

Comment:
I wasn't able to get this to render, I got CORS errors instead:

![Screenshot from 2021-03-05 11-09-12](https://user-images.githubusercontent.com/856138/110107778-41ae1100-7da3-11eb-9281-ae2414288a8a.png)

Reply:
Interesting! Works well for me. I will look into that.

Reply:
My fault, developer config to blame!